scilpy/io/utils.py

```diff
@@ -13,6 +13,35 @@
 from scilpy.utils.bvec_bval_tools import DEFAULT_B0_THRESHOLD
 
 
+def write_bvals_bvecs(bvals, bvecs, outpath=None, prefix=''):
```

Comment:
Can you remove this function and save the bval and bvec files like in scil_extract_dwi_shell.py.

You can add the saving directly in scil_reorder_dwi_philips.py